src/field-value.js

```diff
@@ -20,13 +20,13 @@
  * Sentinel value for a field delete.
  *
  */
-const DELETE_SENTINEL = {};
+let DELETE_SENTINEL;
```

Comment:
Why was this necessary?

Reply:
By making these objects constructed with `new`, they end up failing the isPlainObject() check and don't get persisted as `{}` in other clients.

Reply:
Ok, but you can still keep them as constants right?

Reply:
Once we make this a const, we can't re-assign the value. Since this value is used in the class itself, we also have to declare it before the class.